<launch>
    <node pkg="ABC_EqF" type="ABC_EqF_node.py" name="ABCEqF" output="screen">

        <!-- Input measurement parameters -->
        <param name="gyro_measurement_standard_deviation"   value="0.01"/>
        <param name="gyro_bias_standard_deviation"          value="0.0001"/>

        <!-- Sensors -->
        <rosparam param="sensors">
            - topic: "sensor0"
              type: "calibrated"
              dir: [1, 0, 0]
              std: [0.15, 0.15, 0.15]
              cal: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
              id: -1
            - topic: "sensor1"
              type: "uncalibrated"
              dir: [0, 0, 1]
              std: [0.15, 0.15, 0.15]
              cal: []
              id: 0
        </rosparam>

        <!-- Initial standard deviation -->
        <rosparam param="initial_standard_deviations">
            core: 1.0
            cal: 1.0
        </rosparam>

    </node>
</launch>
